packages/hunspell.rb

```diff
@@ -37,14 +31,18 @@ def self.build
 
   def self.install
     system "make", "DESTDIR=#{CREW_DEST_DIR}", "install"
+  end
+
+  def self.postinstall
     system 'git clone -b libreoffice-6-1 --depth 1 git://anongit.freedesktop.org/libreoffice/dictionaries.git'
     system "install -Dm644 dictionaries/en/en_US.aff #{CREW_DEST_DIR}$HOME/Library/Spelling/en_US.aff"
     system "install -Dm644 dictionaries/en/en_US.dic #{CREW_DEST_DIR}$HOME/Library/Spelling/en_US.dic"
```

Comment:
This package was created before we added the constant but you could replace `#{CREW_DEST_DIR}$HOME` with `#{CREW_DEST_HOME}`.

Reply:
That said, we still want the `system "install ...` lines in the install section.  We could add something like this in the postinstall section:
```
system 'git clone -b libreoffice-6-1 --depth 1 git://anongit.freedesktop.org/libreoffice/dictionaries.git'
FileUtils.c 'dictionaries/en/*', "#{HOME}/Library/Spelling/"
FileUtils.c 'dictionaries/fr/*', "#{HOME}/Library/Spelling/"
...
```

Reply:
> That said, we still want the `system "install ...` lines in the install section. 

Why do you need it in the install section ? Not clear to me

> We could add something like this in the postinstall section:
> 
> ```
> system 'git clone -b libreoffice-6-1 --depth 1 git://anongit.freedesktop.org/libreoffice/dictionaries.git'
> FileUtils.cp Dir.glob('dictionaries/en/*'), "#{HOME}/Library/Spelling/"
> FileUtils.cp Dir.glob('dictionaries/fr/*'), "#{HOME}/Library/Spelling/"
> ...
> FileUtils.rm_rf 'dictionaries/'
> ```

Why not simply replace actual lines by these ones ? 

Why removing the `dictionaries` folder ? Isn't it automatically placed in tmp and then deleted once postinstall ends like it is done with install ?

Reply:
The postinstall section can be run on demand after the package is installed.  We want the `system "install ...` lines in the install section to add to the package metadata.  Otherwise, the package would not track these files for removal when the package is uninstalled.

Reply:
Fine ! Do the latest commit fulfills these "requirements" ?